### PR TITLE
Using stable packages

### PR DESCRIPTION
**Pull request by challgren**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,10 +19,10 @@
 	],
 	"require": {
 		"php": ">=7.2",
-		"cakephp/cakephp": "4.x-dev as 4.0.0"
+		"cakephp/cakephp": "^4.0"
 	},
 	"require-dev": {
-		"cakephp/migrations": "4.x-dev",
+		"cakephp/migrations": "3.0.0-beta2",
```
Comment (dereuromark) [inline composer.json]: Shouldnt ^3.0.0 suffice?

**Comment by challgren** [comment]
Oh I wasn't sure

**Comment by dereuromark** [comment]
"minimum-stability": "dev",

File: composer.json (hunk 1 above)
Comment (challgren) [reply composer.json]: Ahh missed that

**Pull request merged**